packages/appeals-service-api/src/lib/config.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
       path: process.env.DOCS_API_PATH || path.join(__dirname, '..', '..', 'api'),
     },
   },
+  documents: {
```

Comment:
This isn't configured in K8S for appeals-service. I'll add it in

Reply:
Thanks.